01-introduction.md

```diff
@@ -74,11 +75,19 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn to read. While there is some evidence that proficient, adult readers
+identify whole word patterns, the scientific consensus among linguists,
+psychologists and neuroscientists is that in order for a child's brain to learn
+how to read, it is required to first make out the mapping of sounds to letters.
+
+So while whole-word instruction takes an optimistic and student-centered view on
+learning, encouraging enjoyment in learning as a key element in the process , it
```

Comment:
“ , ” → “, ”.  And I'd be tempted to use some em dashes (—) here to avoid layers of commas :p.